listenbrainz/webserver/views/status_api.py

```diff
@@ -64,3 +79,163 @@ def _convert_timestamp_to_string_dump_format(timestamp):
         String of the format "20190625-170100"
     """
     return timestamp.strftime("%Y%m%d-%H%M%S")
+
+
+def get_stats_timestamp():
+    """ Check to see when statistics were last generated for a "random" user. Returns unix epoch timestamp"""
+
+    cache_key = STATUS_PREFIX + ".stats-timestamp"
+    last_updated = cache.get(cache_key)
+    if last_updated is None:
+        url = current_app.config["API_URL"] + "/1/stats/user/rob/artists"
```

Comment:
Probably call couchdb directly here.

Reply:
Done.

Reply:
Did you forget to commit some changes because I still see the API call here?